src/components/GalleryEditor/getInitialCollectionsFromServer.ts

```diff
@@ -13,47 +13,34 @@ import { generate12DigitId } from '~/utils/generate12DigitId';
 import { removeNullValues } from '~/utils/removeNullValues';
 
 export function getInitialCollectionsFromServer(
-  queryRef: getInitialCollectionsFromServerFragment$key
+  galleryRef: getInitialCollectionsFromServerFragment$key
 ): CollectionMap {
-  const query = readInlineData(
+  const gallery = readInlineData(
     graphql`
-      fragment getInitialCollectionsFromServerFragment on Query @inline {
-        galleryById(id: $galleryId) {
-          __typename
-          ... on Gallery {
-            dbid
-            collections {
+      fragment getInitialCollectionsFromServerFragment on Gallery @inline {
```

Comment:
this whole block is me just making the fragment on the Gallery type instead of Query so we can reuse it in the mutation response